extra/2-piping.js

```diff
@@ -16,30 +16,31 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(firstNum, secondNum) {
+  return firstNum + secondNum;
 }
 
-function multiply() {
-
+function multiply(firstNum, secondNum) {
+  return firstNum * secondNum;
 }
 
-function format() {
-
+function format(num) {
+  return `£${num}`;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+let badCode = format(multiply(add(startingValue, 10), 2));
 
 /* BETTER PRACTICE */
-
-let goodCode =
+let result = add(startingValue, 10);
+result = multiply(result, 2);
+let goodCode = format(result);
```

Comment:
Yup this is certainly better; one more  improvement that could be made here though is to use `const` instead to declare multiple variables e.g.
```
const addResult = add(startingValue, 10)
const multiplyResult = multiply(addResult, 2)
const formattedResult = format(multiplyResult)
```
A nice rule of thumb to go by with code-style is to always err on the more verbose side (i.e. it's almost always better to be more explicit about your intentions). The code will give the same result either way, so all your code-style is for is for other developers who'll look at your code, and in that case it's better to make your intentions very obvious.